lib/cluster/s3_distribution_bucket.rb

```diff
@@ -5,6 +5,12 @@ def self.find_or_create(bucket_name)
       return bucket if bucket
 
       S3Bucket.create(name: bucket_name)
+      s3_client.put_public_access_block({
+        bucket: bucket_name,
+        public_access_block_configuration: {
+          block_public_policy: false
+        }
+      })
```

Comment:
According to the following AWS docs, this defaults to false anyway. I'm curious why it was added here. Is it because it's the distribution bucket? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_public_access_block

Reply:
This was related to comment that I don't see anymore and I don't know what I had in mind from this comment, I'm resolving it.

Reply:
Hi Karen,

Not sure what happened to your earlier comment. 

Regarding the public access block setting, it is false by default, but at some level in our accounts it is set to true. I double checked this by creating a quick test bucket:

```
$ aws s3api get-public-access-block --bucket jluker-test-block-public-access-setting
{
    "PublicAccessBlockConfiguration": {
        "BlockPublicAcls": true,
        "IgnorePublicAcls": true,
        "BlockPublicPolicy": true,
        "RestrictPublicBuckets": true
    }
}
```